deeppavlov_agent/core/state_manager.py

```diff
@@ -75,7 +80,7 @@ async def add_failure_bot_utterance(self, dialog: Dialog, payload: Dict, label:
     async def save_dialog(self, dialog: Dialog, payload: Dict, label: str, **kwargs) -> None:
         await dialog.save(self._db)
 
-    async def get_or_create_dialog_by_tg_id(self, user_telegram_id, channel_type):
+    async def get_or_create_dialog(self, user_telegram_id, channel_type, **kwargs):
         return await Dialog.get_or_create_by_ext_id(self._db, user_telegram_id, channel_type)
```

Comment:
Do you need `**kwargs` here?

Reply:
Otherwise, it will fail.
https://github.com/deepmipt/dp-agent/blob/453a903f7980b2619f86c37bae410346093e5315/deeppavlov_agent/core/agent.py#L32 
It is needed for the custom state manager to operate properly.